sktime/forecasting/compose/tests/test_reduce_global.py

```diff
@@ -175,3 +184,54 @@ def test_list_reduction(y, index_names):
     forecaster2.fit(y, fh=[1, 2])
     y_pred = forecaster2.predict(fh=[1, 2, 12])
     check_eval(y_pred.index.names, index_names)
+
+
+def test_equality_transfo_nontranso():
+    """Test that recursive reducers return same results for global / local forecasts."""
+    ensemblealgos = [
+        RandomForestRegressor(),
+        GradientBoostingRegressor(),
+        HistGradientBoostingRegressor(),
+    ]
+
+    y = load_airline()
+    y_train, y_test = temporal_train_test_split(y, test_size=30)
+    fh = ForecastingHorizon(y_test.index, is_relative=False)
+
+    lag_vec = [i for i in range(1, 13)]
+    lag_vec.reverse()
```

Comment:
Instead of reverse we can use range functionality `lag_vec = [i for i in range(12, 0, -1)]` in one line. This is nit-picking, so I'm fine if we leave it as is too :)

Reply:
it hurts my brain that 13 becomes 12...

Reply:
I come from "if we can make it the way we want it from the beginning, why not".